### ISSUE TITLE
Semantic highlighting broken

### ISSUE THREAD

**Issue by parched** [microsoft/pylance-release#3667]
## Environment data

-   Language Server version: Pylance language server 2022.11.32 (pyright 45344736)
-   OS and version: Windows 10
-   Python version (& distribution if applicable, e.g. Anaconda): python 3.10

## Code Snippet

![image](https://user-images.githubusercontent.com/5975405/203463788-07c71ebb-fde0-474e-ab61-c906d8717706.png)

It looks like it's using the wrong line
![image](https://user-images.githubusercontent.com/5975405/203464685-8c46cd5e-08a5-4a9c-b626-5873e9e405b0.png)

is actually the highlighting for this line in the same file
![image](https://user-images.githubusercontent.com/5975405/203464790-0bea65a4-1e6d-424f-b2f3-06548d7058d1.png)

This part of large multi root workspace so I don't have a small reproducer yet unfortunately.

These are the logs that refer to the file, is there anything else in the logs I should look out for?
```
[Info  - 4:06:32 pm] (17668) [BG(18)] Long operation: getSemanticTokens full at path/to/file.py (5427ms)
[Info  - 4:06:44 pm] (17668) [BG(18)] Long operation: checking: path/to/file.py (2395ms)
[Info  - 4:06:44 pm] (17668) [BG(18)] Long operation: analyzing: path/to/file.py (2656ms)
(17668) [BG(12)] analyzing: path/to/file.py ...
(17668) [BG(12)]   parsing: path/to/file.py (55ms)
(17668) [BG(12)]   binding: path/to/file.py (1ms)
(17668) [BG(12)]   checking: path/to/file.py ...
(17668) [BG(12)]   checking: path/to/file.py (1015ms)
(17668) [BG(12)] analyzing: path/to/file.py (1684ms)
(17668) [BG(18)] analyzing: path/to/file.py ...
(17668) [BG(18)]   parsing: path/to/file.py (160ms)
(17668) [BG(18)]   binding: path/to/file.py (3ms)
(17668) [BG(18)]   checking: path/to/file.py ...
(17668) [BG(12)] indexing: path/to/file.py [found 6] (0ms)
(17668) Indexing Done: path/to/file.py
(17668) [FG] parsing: path/to/file.py (66ms)
(17668) [FG] binding: path/to/file.py (2ms)
(17668) [BG(18)]   checking: path/to/file.py (2959ms)
[Info  - 4:16:21 pm] (17668) [BG(18)] Long operation: checking: path/to/file.py (2959ms)
(17668) [BG(18)] analyzing: path/to/file.py (3286ms)
[Info  - 4:16:21 pm] (17668) [BG(18)] Long operation: analyzing: path/to/file.py (3286ms)
```

**Comment by parched** [comment]
Potentially duplicate of https://github.com/microsoft/pylance-release/issues/3649. Rolling back to v2022.11.20 seems to have fixed it.

**Comment by bschnurr** [comment]
This might be related to this issue. fix coming in today's pre-release.
https://github.com/microsoft/pylance-release/issues/3652

**Comment by heejaechang** [comment]
he meant next release which is next week.

**Comment by rchiodo** [comment]
This issue has been fixed in prerelease version 2022.11.41, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#20221141-30-November-2022-PreRelease)
